src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioErrorReportingService.cs

```diff
@@ -31,43 +30,72 @@ internal partial class VisualStudioErrorReportingService : IErrorReportingServic
             _listener = listener;
         }
 
-        public void ShowErrorInfo(string message, params ErrorReportingUI[] items)
+        public void ShowErrorInfoInActiveView(string message, params ErrorReportingUI[] items)
```

Comment:
@jmarolf: should we just have everything use the global one? We were only using active view because we didn't know the global one existed or it didn't exist at the time?